api/rest/new_mock_client_test.go

```diff
@@ -0,0 +1,31 @@
+package rest
+
+import "context"
+
+type AlternativeMockClient struct {
```

Comment:
Why the `Alternative` here?

Reply:
We already have `MockClient` I don't know how to name this one. Maybe I can alter the code to use teststify

Reply:
Renamed